### PR TITLE
Tweak installer to meet accessibility standards

### PR DESCRIPTION
**Pull request by thcrain-msft**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

The installer does not meet some accessibility requirements, when it comes to things like contrast and screen reading. This PR adds some tweaks to meet those requirements.

###### Change Log  

- Tweak and remove sections that are tricky for speakup to read in a sensible manner
- Adjust contrast in the exit modal

###### Does this affect the toolchain?  

NO

###### Test Methodology

- Local test of image

### CODE REVIEW

File: toolkit/tools/imagegen/attendedinstaller/attendedinstaller.go [hunk 1]
```diff
@@ -220,6 +234,10 @@ func (ai *AttendedInstaller) globalInputCapture(event *tcell.EventKey) *tcell.Ev
 }
 
 func (ai *AttendedInstaller) initializeUI() (err error) {
+	ai.keyboard, err = speakuputils.CreateVirtualKeyboard()
+	if err != nil {
+		logger.Log.Warnf("Failed to initialize virtual keyboard via uinput")
```
Comment (schmittjoseph) [inline toolkit/tools/imagegen/attendedinstaller/attendedinstaller.go:239]: Is this error fatal?

Comment (thcrain-msft) [reply toolkit/tools/imagegen/attendedinstaller/attendedinstaller.go:239]: No- the failure to initialize the virtual keyboard just means the text-to-speech buffer won't be automatically cleared after each screen. This results in a slightly degraded experience for a blind/visually impaired user, but it is still usable and is no reason to kill the installer.

Comment (schmittjoseph) [reply toolkit/tools/imagegen/attendedinstaller/attendedinstaller.go:239]: Good to know. In that case I'd recommend two things:
1. Add a comment stating this is a non-fatal error.
2. Either clear `err` here or use a temporary err variable instead to ensure the error never gets returned by the main routine in the future if a refactor happens.

Comment (thcrain-msft) [reply toolkit/tools/imagegen/attendedinstaller/attendedinstaller.go:242]: Addressed both points in latest commit.

**Review by schmittjoseph — APPROVED** [review]
I think the `toolkit/tools/cgmanifest.json` file needs to be updated, but otherwise it looks good to me

**Pull request merged**